tailwind.config.js

```diff
@@ -11,5 +11,10 @@ bloomTheme.purge = [
   "app/javascript/**/*.tsx",
   "node_modules/@bloom-housing/ui-components/src/**/*.tsx",
 ]
-
+bloomTheme.theme.fontSize["4xl"] = ['2rem', {
```

Comment:
this is due to an issue our designer found. I wonder if it is something you'd want on bloom too?

Reply:
@emilyjablonski ping again on this question and [this one](https://github.com/SFDigitalServices/sf-dahlia-web/pull/1552/files#r774255933)